File: linguagens/c#/algoritmos/quest_02.cs
// Curso: Algoritmos
// Professor: Gustavo Guanabara
// Autor: Franciney de Araujo
// Data: 28/08/2026
// faça um programa que leia o nome de uma pessoa e mostre uma mensagem de boas-vindas para ela:"

// ALGORITMO
// leia(nome)
// leia(salario)
// escreva "Meu nome é (nome), meu salario é (salario)"



class Program
{

    static void Main()
    {
        // leia(nome)
        string nome = "Ana";

        // leia(salario)
        double salario = 2500.50;

        // escreva "Meu nome é (nome), meu salario é (salario)"
        Console.WriteLine($"Menu nome é {nome}, meu salario é {salario}");
    }
}

File: linguagens/java/algoritmo/quest_02.java
// Curso: Algoritmos
// Professor: Gustavo Guanabara
// Autor: Franciney de Araujo
// Data: 28/08/2026
// faça um programa que leia o nome de uma pessoa e mostre uma mensagem de boas-vindas para ela:"


// bibliote para o objeto scanner
import java.util.Scanner;

class quest_02 {
    public static void main(String[] args) {

        // Declaração
        String nome = "";

        // Cria o Scanner
        Scanner scanner = new Scanner(System.in);

        // Entrada
        System.out.print("Digite seu nome: ");
        nome = scanner.nextLine();

        // Saída
        System.out.println("Seja bem vindo " + nome);

        scanner.close();
    }
}